tests/macsec/test_static_key_ciphers.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+import logging
+import random
+import time
+import json
+import os
+
+from .macsec_config_helper import setup_macsec_configuration, cleanup_macsec_configuration
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.macsec_required,
+    pytest.mark.topology("t2"),
+]
+
+
+def macsec_check(host, cli_options, int, neighv4, neighv6, macsec=True, cipher=None):
+    if macsec:
+        # Verify Macsec Status Between Neighbors is up
+        output = host.shell('sonic-cfggen{} -d -v PORT.{}'.format(cli_options, int),
+                            module_ignore_errors=True)['stdout']
+        logger.debug(output)
+        logger.debug("'macsec': '{}'".format(cipher))
+        assert "'macsec': '{}'".format(cipher) in output
+        # Verify WPA Supplicant process is running on specific port
+        output = host.shell('ps aux | grep "USER\\|wpa_supplicant"', module_ignore_errors=True)['stdout']
+        logger.debug(output)
+        assert int in output
+        # Verify macsec is enabled on port and session is up and established
+        output = host.shell("show macsec{} {}".format(cli_options, int), module_ignore_errors=True)
+        output = output['stdout'].split("\n")
+        logger.debug(output)
+        assert "enable                 true" in output[3]
+
+    # Verify BGP Between Neighbors is established
+    output = host.shell("show ip bgp neighbor {}".format(neighv4))['stdout']
+    logger.debug("BGP v4: {}".format(output))
+    assert "BGP state = Established" in output
+    output = host.shell("show ipv6 bgp neighbor {}".format(neighv6))['stdout']
+    logger.debug("BGP v6: {}".format(output))
+    assert "BGP state = Established" in output
+
+
+def test_static_key_ciphers(duthost, nbrhosts, request, profile_name, tbinfo, ctrl_links, rekey_period):
```

Comment:
duthost fixture may get you a T2 supervisor. Change to use the enum_rand_one_per_hwsku_frontend_hostname fixture to get duthost

Reply:
I have updated the test to use this fixture.